code/modules/antagonists/blob/blob_abilities.dm

```diff
@@ -227,6 +227,10 @@
 						boutput(owner, "<span class='alert'>You are being watched.</span>")
 						return
 
+		if (owner)
+			if (tgui_alert(owner,"Would you like to deploy your nucleus?","Deploy Nucleus?",list("Yes","No")) != "Yes")
+				return TRUE
+
```

Comment:
Again, put it below the tutorial check so you don't get a confirmation prompt if you can't use the ability